packages/components/atoms/f-image-tile/test-utils/component-objects/f-imageTile.component.js

```diff
@@ -0,0 +1,26 @@
+const Page = require('@justeat/f-wdio-utils/src/page.object');
+const { COMPONENT } = require('./f-imageTile-selectors')
+
+module.exports = class ImageTile extends Page {
+    constructor () {
+        super('atom', 'image-tile-component');
+    }
+
+    get component () { return $(COMPONENT); }
```

Comment:
I think this complains with an eslint issue.
```suggestion
    get component () {
        // eslint-disable-next-line no-undef
        return $(COMPONENT);
    }
```

Reply:
COMPONENT is defined at the top of the file. Is this still an issue?

Reply:
Yes but I think as it is used within the boundary of the a class JS expect it to be in the class and referenced using the term `this.` but as JS also allows us to keep things private and outside of the boundary of the class it goes against its own rules :) ... so lint then complains

Reply:
updated